projects/plugins/jetpack/extensions/shared/external-media/media-service/index.ts

```diff
@@ -177,24 +195,46 @@ export const getPexelsMediaCategory = () =>
 	);
 
 /**
- * Checks if GooglePhotos is connected and calls the callback with the response.
+ * Checks if a given MediaSource is authenticated in the store.
  *
- * @param {ConnectedMediaSourceCallback} isConnectedCallback
- * @returns {void}
+ * @param {MediaSource} source - MediaSource to check.
+ * @returns {boolean} True if the MediaSource is authenticated false otherwise.
  */
-export const isGooglePhotosConnected = ( isConnectedCallback: ConnectedMediaSourceCallback ) =>
-	isMediaSourceConnected( MediaSource.GooglePhotos, isConnectedCallback );
+const isMediaSourceAuthenticated = ( source: MediaSource ) =>
```

Comment:
Nitpick (non-blocking): It's not clear to me what's the difference between `isMediaSourceConnected` and `isMediaSourceAuthenticated`, since the authentication is the same as a connection in the context of a external media source.

Reply:
Changed to a name that's a bit clearer about the intent of the use case.